ios-target/ViewModel/Home/HomeViewModel.swift

```diff
@@ -97,27 +97,35 @@ class HomeViewModel {
     state = .loading
     TargetAPI.getTargets({ [weak self] targets in
       self?.targets = targets
+      self?.addAnnotations()
       self?.state = .idle
       }, failure: { [weak self] error in
         self?.state = .error(error.localizedDescription)
     })
   }
   
-  // TODO:
-//  func addAnnotations() {
-//    let appleParkAnnotation = MKPointAnnotation()
-//    appleParkAnnotation.title = "Apple Park"
-//    appleParkAnnotation.coordinate = CLLocationCoordinate2D(latitude: 37.333303, longitude: -122.011252)
-//
-//    delegate?.addAnnotations(annotations: [appleParkAnnotation])
-//  }
+  func addAnnotations() {
+    var annontations: [TargetAnnotation] = []
+    var circleOverlays: [TargetCircle] = []
+    targets.forEach { target in
+      let coordinates = CLLocationCoordinate2D(latitude: target.latitude, longitude: target.longitude)
+      let annotation = TargetAnnotation(coordinate: coordinates,
+                                        title: target.title,
+                                        subtitle: "",
+                                        type: .travel,
```

Comment:
Why always travel type? Doesn't Targets have a type or a topic?

Reply:
yes, haven't implemented that yet 😅